src/main/java/com/authlete/common/dto/IntrospectionResponse.java

```diff
@@ -500,8 +524,6 @@ public enum Action
      *
      * @since 3.41
      * @since Authlete 2.1.24
-     * @since Authlete 2.2.36
```

Comment:
This line has a meeting, but if multiple `@since Authlete ...` lines may cause errors in your script, it may be okay to remove this line.

Reply:
My script manages it. If there are other similar cases, it's best to have them handled.